### ISSUE TITLE
Batch Nomalization 

### ISSUE THREAD

**Issue by xuexingyu24** [jacobgil/pytorch-pruning#35]
hi jacobgil 

It is a very nice work, Just wondering if you have any ideas or suggestions for Batch normalization which contains the info from upstream conv layer ?   We need to handle the scale and shift if any filters are pruned.
